src/components/home/elements/SplitTokenOrSupersede.vue

```diff
@@ -551,19 +693,21 @@ export default {
                 } else if (opc !== OPC_ACCOUNT) {
                     this.paused = false
                     this.qrErrMsg = 'Wrong operation code in QR code.'
-                } else if (!this.isValidIssuer(this.newIssuer)) {
+                } else if (!this.isValidIssuer) {
                     this.paused = false
                     this.qrErrMsg = 'Invalid address of new issuer.'
                 } else {
                     this.qrErrMsg = void 0
                 }
             } catch (e) {
-                if (this.isValidIssuer(decodeString)) {
-                    this.newIssuer = decodeString
+                this.newIssuer = decodeString
+                if (this.isValidIssuer) {
+                    newIssuer = decodeString
```

Comment:
The same here. The code can be simplify with use `this.newIssuer` only. No need local variable  `newIssuer`.